impeller/entity/contents/texture_contents.cc

```diff
@@ -146,12 +146,13 @@ bool TextureContents::Render(const ContentContext& renderer,
   frame_info.texture_sampler_y_coord_scale = texture_->GetYCoordScale();
   frame_info.alpha = capture.AddScalar("Alpha", GetOpacity());
 
-  Command cmd;
+#ifdef IMPELLER_DEBUG
   if (label_.empty()) {
-    DEBUG_COMMAND_INFO(cmd, "Texture Fill");
+    pass.SetCommandLabel("Texture Fill");
```

Comment:
You just set it to the unnamed label in non debug builds.

Reply:
It will get dropped anyway, because SetCommandLabel will no-op